src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int numberOfBall = new Random().nextInt(100);
+        Ball ball = new Ball();
+        ball.SetColor(colorSupplier.toString());
```

Comment:
You are calling `toString()` on the `ColorSupplier` instance. This is not correct. `ColorSupplier` should have a method that returns a random color. You need to call this method instead of `toString()`. Also, the method name `SetColor` does not follow Java naming conventions. It should be `setColor`.